plugins/base/src/main/java/edu/wpi/first/shuffleboard/plugin/base/BasePluginVersion.java

```diff
@@ -0,0 +1,18 @@
+package edu.wpi.first.shuffleboard.plugin.base;
+
+/**
+ * Autogenerated version class containing the version of the Base plugin.
+ *
+ * <p><strong>DO NOT MODIFY THIS CLASS</strong>
+ */
+public final class BasePluginVersion {
+
+  /**
+   * The version of the Base plugin.
+   */
+  public static final String VERSION = "1.5.0";
+
+  private BasePluginVersion() {
+    throw new UnsupportedOperationException();
+  }
+}
```

Comment:
EOL @ EOF

Reply:
Irrelevant for an autogenerated file IMO

Reply:
We should edit line six to say that it is autogenerated.